lib/accesscontrol.rb

```diff
@@ -3,10 +3,14 @@
 
 module AccessControl
 
-  unless defined?(CouldNotGrantError) == "constant" && CouldNotGrantError.class == Class
-    CouldNotGrantError = Class.new(StandardError)
+  unless defined?(GrantError) == "constant" && GrantError.class == Class
+    GrantError = Class.new(StandardError)
```

Comment:
@ehourigan I'm a little curious on this one... why not just define grant error as a class that inherits from StandardError and let ruby handle the autoloading like:
```Ruby
class GrantError < StandardError
end
```

Reply:
I think that should work, but if not they might need their own file named after them so the autoloading can find them... some thing like in `access_control/grant_error.rb`
```Ruby
module AccessControl
  class GrantError < StandardError
  end
end
```

Reply:
This is a signature I borrowed from `acts_as_archival` (sorry - that comment was in another PR)

Which means to say I don't have a strong opinion on how to load the `StandardError` class

Reply:
Ah sweet, I see :+1: personally I would prefer the way I described for clarity... but I don't feel strongly either way; I was just interested in the story of how it came about. I'll leave it totally up to you and Aaron as far as what and if we want to do anything with that :-)

Reply:
For simple classes that don't do anything beyond what the inherited class does, we have historically gone with the style done here. See for example: https://github.com/lessonly/lessonly/blob/master/app/controllers/application_controller.rb#L20